src/Services/CreateExport.php

```diff
@@ -27,7 +32,13 @@ public function run(): Export
             'type' => $this->dto->getType(),
             'generator_type' => $this->dto->getGeneratorType(),
             'deliveries' => $this->dto->getDeliveries(),
-            'args' => $this->dto->getArgs(),
+            'args' => array_merge(
+                $dtoArgs,
+                [
+                    'dateTo' => $dateTo,
+                    'dateFrom' => $dateFrom,
+                ]
+            )
```

Comment:
The args hold a little more information, like companies. e.g. `{"companies":[82211],"dateFrom":"2022-08-08","dateTo":"2022-08-08"}`. The issue was that they are indeed passed in but laravel does magic with timezones and attempts to convert the dates to Carbon instance and then saving as datetime timezone which caused the actual bug. More details https://stackoverflow.com/questions/60649769/wrong-timezone-in-laravel-7-after-date-serialization

GraphQL handles the date range and validation so there is always a date, however it can be either as string or as carbon instance, so this makes sure it's safe.